src/js/components/ErrorableCheckbox.jsx

```diff
@@ -46,7 +46,7 @@ class ErrorableCheckbox extends React.Component {
     return (
       <div className={className}>
         <input
-          autoComplete="false"
```

Comment:
Is the `|| null` needed for these cases? The `null` is not treated as a missing value so if there is an `ErrorableCheckbox.defaultProps` React won't use the `autoComplete` value in it. So at the very least this should be changed to `undefined` but I think we may not need it at all. If there is no `options.autocomplete` we get `undefined` for free and I don't think we care about other falsy values like `0`, `NaN`, `false`, or `""` because people shouldn't be passing them.

Reply:
I think I understand. 

It's better to use the `propTypes` and `defaultProps` to convey what are required, optional, and default values in react components. 

So in this case, `options` object should be added as an optional prop to `propTypes` with no `defaultProps` values and we should remove `|| null`.